app/src/main/res/values/example_titles.xml

```diff
@@ -17,7 +17,7 @@
     <string name="activity_map_viewpager">ViewPager</string>
     <string name="activity_map_multi">Multi map fragments</string>
     <string name="activity_debug_mode">Debug mode</string>
-    <string name="activity_fill_extrusion">Display building extrusions</string>
+    <string name="activity_fill_extrusion">Display 3D buildings</string>
```

Comment:
Aligns with iOS and IMO is easier to understand at first glance. (The term "extrusion" is still used in the description to tie back to the technical language.)